packages/adblocker-electron/src/preload_path-cjs.cts

```diff
@@ -0,0 +1,3 @@
+import { resolve } from 'node:path';
```

Comment:
This file was meant to be used by `tshy` but is currently not in use. Its safe took keep as a reference.

Reply:
https://github.com/isaacs/tshy?tab=readme-ov-file#commonjs-dialect-polyfills